lib/dealership.rb

```diff
@@ -0,0 +1,22 @@
+class Dealership
+  attr_accessor :name, :address, :inventory, :inventory_count, :car
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
```

Comment:
So I think this is a way that you could do the `inventory_count` like this. Here is another possible solution:
```
def inventory_count
    @inventory.length
  end
```